### PR TITLE
Dev/ar/redev

### PR DESCRIPTION
**Pull request by AdrienR**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 .vagrant
 .idea
 venv
+*pyc
```
Comment (jlejeune) [inline .gitignore]: *.pyc instead of *pyc no?

File: OVHJabberBot/commands/baguette.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def ask_baguette():
     """ Demande aux gens s'ils veulent une baguette """
     global round
     orders = Order.objects()
-    notifs = Notif.objects(times__gt=round)
+    notifs = Notif.get_all(times_gt=round)
 
     results = [user.name for user in notifs if user not in [order.name for order in orders] and user not in non_list]
```
Comment (arnaudmorin) [inline OVHJabberBot/commands/baguette.py]: ta un haricot ici

**Review by arnaudmorin — CHANGES_REQUESTED** [review]
haricot

**Review by arnaudmorin — APPROVED** [review]
haricot proof

**Pull request merged**